tests/Microsoft.Identity.Test.Integration.netcore/SeleniumTests/DeviceCodeFlowIntegrationTest.cs

```diff
@@ -62,19 +62,6 @@ public async Task ArlingtonDeviceCodeFlowTestAsync()
             await AcquireTokenWithDeviceCodeFlowAsync(labResponse, "aad user").ConfigureAwait(false);
         }
 
-        [TestMethod]
-        [Timeout(2 * 60 * 1000)] // 2 min timeout
-        [TestCategory(TestCategories.ADFS)]
-#if IGNORE_FEDERATED
-        [Ignore]
-#endif
-        public async Task DeviceCodeFlowAdfsTestAsync()
-        {
-            LabResponse labResponse = await LabUserHelper.GetAdfsUserAsync(FederationProvider.ADFSv2019, true).ConfigureAwait(false);
```

Comment:
There is already a Arlington ADFS user test, this `GetAdfsUserAsync(FederationProvider.ADFSv2019, true)` still invokes the same user. Two different tests using the same user to test the flow. This is not needed.